cwt/draft-ietf-ace-cbor-web-token.xml

```diff
@@ -157,7 +157,7 @@
             The "Type6NumericDate" term has the same meaning, syntax, and
             processing rules as the "NumericDate" term defined in Section 2
             of JWT <xref target="RFC7519" />, except that Type6NumericDate uses
-            CBOR major type 6, with tag value 1, instead of a numeric JSON
+            CBOR major type 6.1, instead of a numeric JSON
```

Comment:
This is better, but I think it would be even clearer if, upon first use, we spell this out as "a CBOR number tagged with tag 6.1 (Epoch-based date/time)".